glide.lock

```diff
@@ -0,0 +1,51 @@
+hash: db55a031aaa2f352fa5e9e4fda871039afb80e383a57fc77e4b35114d47cca8a
```

Comment:
I see. That does sound useful. Also [attributed to you](http://engineeredweb.com/blog/2014/glide-go-package-management/). Nice. There's a theme here.:0)

It looks/sounds a bit like npm, which I like a lot. Was that the mental model for it?

How does it compare with godeps/vendoring?